openedx/features/wikimedia_features/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
```

Comment:
wikimedia_features will be a container, in which we'll have dashboard app.

Reply:
sure